bokeh/cli/core.py

```diff
@@ -0,0 +1,274 @@
+from __future__ import print_function
+
+import time
+try:
+    import urllib.request as urllib2
+except ImportError:
+    import urllib2
```

Comment:
you can use six here too... https://pythonhosted.org/six/#module-six.moves.urllib.request